lib/board.rb

```diff
@@ -0,0 +1,69 @@
+class Board
+  attr_accessor :cells
+
+  def reset!
+    # can reset the state of the cells in the board
+    # sets the cells of the board to a 9 element array of " "
+    @cells = Array.new(9, " ")
+  end
+  def initialize
+    # sets the cells of the board to a 9 element array of " "
+    @cells = Array.new(9, " ")
+  end
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.